api/mapping/views.py

```diff
@@ -591,6 +591,27 @@ class ScanReportConceptViewSet(viewsets.ModelViewSet):
     serializer_class = ScanReportConceptSerializer
 
     def create(self, request, *args, **kwargs):
+        body = request.data
+        concept = ScanReportConcept.objects.filter(
+            concept=body["concept"],
+            object_id=body["object_id"],
+            content_type=body["content_type"],
+        )
+        if concept.count() > 0:
+            print("Can't add multiple concepts of the same id to the same object")
+            response = JsonResponse(
+                {
+                    "status_code": 400,
+                    "ok": False,
+                    "statusText": "Can't add multiple concepts of the same id to the same object",
+                }
+            )
+            response.status_code = 400
+            response.statusText = (
+                "Can't add multiple concepts of the same id to the same object"
+            )
```

Comment:
Looks like this is repeated content that needs to only be set once?

Reply:
The content inside the object is what is returned in the body and what is set on line 609 determines the status code of the overall response. line 610 is not needed though and I have now removed it

Reply:
So just to be clear, line 609 is required even though 604 sets the status_code? They are different?